packages/@aws-cdk/aws-codebuild/lib/report-group.ts

```diff
@@ -93,6 +93,16 @@ export interface ReportGroupProps {
    * @default RemovalPolicy.RETAIN
    */
   readonly removalPolicy?: cdk.RemovalPolicy;
+
+  /**
+   * The type of report group. This can be one of the following values:
+   *
+   * - **TEST** - The report group contains test reports.
+   * - **CODE_COVERAGE** - The report group contains code coverage reports.
+   *
+   * @default - TEST
+   */
+  readonly type?: 'TEST' | 'CODE_COVERAGE'
```

Comment:
Please introduce an enum type

Reply:
Done! I was not sure about the documentation tho. Is this ok? I tried to reuse to text which is in the [CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-reportgroup.html). 
Thanks you for your review! ❤️